src/main/java/com/snowflake/kafka/connector/internal/SnowflakeIngestionServiceV1.java

```diff
@@ -142,7 +142,16 @@ public Map<String, InternalUtils.IngestedFileStatus> readIngestReport(final List
           if (fileStatus.containsKey(file.getPath())) {
             numOfRecords++;
 
-            fileStatus.put(file.getPath(), convertIngestStatus(file.getStatus()));
+            final InternalUtils.IngestedFileStatus ingestionStatus = convertIngestStatus(file.getStatus());
+            fileStatus.put(file.getPath(), ingestionStatus);
+            // Log errors
+            if (InternalUtils.IngestedFileStatus.FAILED.equals(ingestionStatus) || InternalUtils.IngestedFileStatus.PARTIALLY_LOADED.equals(ingestionStatus)) {
+              final String firstError = file.getFirstError() != null ? file.getFirstError() : "NO ERROR";
+              final String firstErrorColumnName = file.getFirstErrorColumnName() != null ? file.getFirstErrorColumnName() : "N/A";
+              final String firstErrorCharacterPos = file.getFirstErrorCharacterPos() != null ? file.getFirstErrorCharacterPos().toString() : "N/A";
+              LOG_WARN_MSG("Failed to load file {}\n\tError: {}\n\tColumn: {}\n\tChar pos: {}",
+                      file.getPath(), firstError, firstErrorColumnName, firstErrorCharacterPos);
```

Comment:
this is potentially logging PII data, lets not do this. lets only print file path, failed file ingestion errors can be further looking into ingest copy history table function. https://docs.snowflake.com/en/sql-reference/functions/copy_history

Reply:
We've had several use cases when there is no error description in the copy history, so I was hoping to find more details in these logs.

Reply:
Applied suggestion.